web/src/scheduled_messages_popover.js

```diff
@@ -165,6 +166,7 @@ export function initialize() {
                 focus_first_send_later_popover_item();
                 send_later_popover_keyboard_toggle = false;
             }
+            compose_recipient.update_compose_draft_count();
```

Comment:
Why does not this need to be called here?

Reply:
Just like we call this anytime the compose box is opened, similarly we call this anytime the popover is shown. As the popover is destroyed on hiding, we need to update the drafts count every time it's shown.